_articles/de/how-to-contribute.md

```diff
@@ -1,220 +1,242 @@
+  so-finden-sie-ein-projekt-zu-dem-sie-beitragen-können: "So finden Sie ein Projekt, zu dem Sie beitragen können"
+  wie-man-einen-beitrag-einreicht: "Wie man einen Beitrag einreicht"
+  was-passiert-nachdem-sie-einen-beitrag-eingereicht-haben: "Was passiert, nachdem Sie einen Beitrag eingereicht haben?"
 order: 1
 image: /assets/images/cards/contribute.png
 related:
   - beginners
   - building
 ---
 
-## Why contribute to open source?
+## Warum eigentlich zu Open-Source-Projekten beitragen?
 
 <aside markdown="1" class="pquote">
   <img src="https://avatars.githubusercontent.com/errietta?s=180" class="pquote-avatar" alt="avatar">
-  Working on \[freenode\] helped me earn many of the skills I later used for my studies in university and my actual job. I think working on open source projects helps me as much as it helps the project!
+  
+  An \[freenode\] zu arbeiten, brachte mir die Fähigkeiten ein, die ich später in meinem Universitätsstudium und meinen Beruf nutzte. Ich denke, an Open-Source zu arbeiten, half mir selbst ebenso sehr wie dem Projekt.
+  
+  _Working on \[freenode\] helped me earn many of the skills I later used for my studies in university and my actual job. I think working on open source projects helps me as much as it helps the project!_
+  
   <p markdown="1" class="pquote-credit">
 — @errietta, ["Why I love contributing to open source software"](https://www.errietta.me/blog/open-source/)
   </p>
 </aside>
 
-Contributing to open source can be a rewarding way to learn, teach, and build experience in just about any skill you can imagine.
+Zu Open Source beizutragen, kann ein lohnender Weg sein, um nahezu alle erdenklichen Fertigkeiten zu erlernen, zu lehren und Erfahrungen zu sammeln.
 
-Why do people contribute to open source? Plenty of reasons!
+Warum tragen Menschen zu Open Source bei? Es gibt viele Gründe!
 
-### Improve existing skills
+### Bestehende Fähigkeiten verbessern
 
-Whether it's coding, user interface design, graphic design, writing, or organizing, if you're looking for practice, there's a task for you on an open source project.
+Ob Programmierung, User Interface Design, Grafikdesign, Schreiben oder Organisieren: Wenn Sie auf der Suche nach Praxiserfahrung sind, werden Sie Aufgaben für sich in einem Open-Source-Projekt finden.
 
-### Meet people who are interested in similar things
+### Treffen Sie Menschen, die an ähnlichen Dingen interessiert sind
 
-Open source projects with warm, welcoming communities keep people coming back for years. Many people form lifelong friendships through their participation in open source, whether it's running into each other at conferences or late night online chats about burritos.
+Open-Source-Projekte mit warmherzigen, einladenden Communities schaffen langfristige Hobbies für viele Leute. Lebenslange Freundschaften können durch ihre Teilnahme an Open Source entstehen, sei es bei Konferenzen oder bei nächtlichen Online-Chats über Lamacuns.
```

Comment:
What is `Lamacuns`? :D

Reply:
A Turkish dish that I'd consider as somewhat equivalent to what burritos are for Americans.

Reply:
looks like it is called `Lahmacun` in German.